packages/compass-aggregations/src/components/pipeline-toolbar/index.spec.tsx

```diff
@@ -0,0 +1,281 @@
+import React from 'react';
+import { render, screen, within, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Provider } from 'react-redux';
+import { expect } from 'chai';
+import type { Store } from 'redux';
+
+import type { RootState } from '../../modules';
+import configureStore from '../../stores/store';
+import Aggregations from '../aggregations';
+import { DATA_SERVICE_CONNECTED } from '../../modules/data-service';
+import { STAGE_ADDED, STAGE_OPERATOR_SELECTED } from '../../modules/pipeline';
+
+const mockDataService = class {
+  aggregate() {
+    return {
+      toArray: () => Promise.resolve([{ id: 1 }]),
+    };
+  }
+};
+
+const initialShowNewToolbarValue =
+  process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR;
+
+describe('PipelineToolbar', function () {
+  describe('renders toolbar', function () {
+    let toolbar: HTMLElement;
+    beforeEach(function () {
+      process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR = 'true';
+      const store = configureStore({});
+      render(
+        <Provider store={store}>
+          <Aggregations />
+        </Provider>
```

Comment:
Seems like these tests test and rely on a lot of things that are outside of the actual toolbar implementation. This means that any changes outside the toolbar will break these tests and it will be hard to debug if something really broke or it is some unrelated changes. If these are new tests that we are adding, shouldn't we focus on testing just the toolbar behavior that we expect here?